src/docs/sdk/features.mdx

```diff
@@ -400,6 +400,8 @@ The `fingerprints` field should be set to `["$operationName", "$operationType",
 
 Ability for the SDK to attach request body to events and triggered during the execution of request.
 
+Attaching request and response bodies should be guarded by `sendDefaultPii` or another flag  to opt-in (whichever makes more sense for the specific SDK).
```

Comment:
@adinauer this is under `Attaching Request Body in Server SDKs` which is already opt-in because `maxRequestBodySize` defaults to `none`.
If this is about the GraphQL integration, let's add this specifically to that section.
Otherwise, adapt this section to Request/Response, and mention that this applies to GraphQL as well.

Reply:
Moved it up